test/ConstantProduct/ConstantProductTestHarness.sol

```diff
@@ -5,32 +5,50 @@ import {BaseComposableCoWTest} from "lib/composable-cow/test/ComposableCoW.base.
 
 import {ConstantProduct, GPv2Order, IERC20} from "src/ConstantProduct.sol";
 import {UniswapV2PriceOracle, IUniswapV2Pair} from "src/oracles/UniswapV2PriceOracle.sol";
+import {ISettlement} from "src/interfaces/ISettlement.sol";
 
 import {Utils} from "test/libraries/Utils.sol";
 
 abstract contract ConstantProductTestHarness is BaseComposableCoWTest {
     address internal orderOwner = Utils.addressFromString("order owner");
+    address internal vaultRelayer = Utils.addressFromString("vault relayer");
     address private USDC = Utils.addressFromString("USDC");
     address private WETH = Utils.addressFromString("WETH");
     address private DEFAULT_PAIR = Utils.addressFromString("default USDC/WETH pair");
     address private DEFAULT_RECEIVER = Utils.addressFromString("default receiver");
+    address private DEFAULT_SOLUTION_SETTLER = Utils.addressFromString("settlement contract");
     bytes32 private DEFAULT_APPDATA = keccak256(bytes("unit test"));
     bytes32 private DEFAULT_COMMITMENT = keccak256(bytes("order hash"));
+    bytes32 private DEFAULT_DOMAIN_SEPARATOR = keccak256(bytes("order hash"));
```

Comment:
```suggestion
    bytes32 private DEFAULT_DOMAIN_SEPARATOR = keccak256(bytes("domain separator hash"));
```

To avoid potential typos in tests as `DEFAULT_COMMITMENT` = `DEFAULT_DOMAIN_SEPARATOR` otherwise.